clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -707,6 +708,9 @@ static void addSanitizers(const Triple &TargetTriple,
       MPM.addPass(createModuleToFunctionPassAdaptor(ThreadSanitizerPass()));
     }
 
+    if (LangOpts.Sanitize.has(SanitizerKind::NumericalStability))
+      MPM.addPass(NumericalStabilitySanitizerPass());
```

Comment:
you may want late to introduce post-instrumentations as on line 693 of msan

Reply:
debugging would be a bit harder - so would prefer to postpone adding them, but yeah, thanks for the suggestion, will keep it in mind.